.github/workflows/enos-ci-bootstrap-ent.yml

```diff
@@ -0,0 +1,44 @@
+name: enos-ci-bootstrap-ent
```

Comment:
We don't need to include ent or hcp stuff in the OSS repo. Better to merge the OSS stuff in here and create another PR for the ent/hcp stuff.

Reply:
I removed the enterprise workflow